modApiServer/src/org/aion/api/server/http/ApiWeb3Aion.java

```diff
@@ -2296,6 +2336,202 @@ public RpcMsg stratum_getHeaderByBlockNumber(Object _params) {
         return new RpcMsg(obj);
     }
 
+    // always gets the latest 20 blocks and transactions
+    private class MinerStatsView {
+        LinkedList<byte[]> hashQueue; // more precisely a dequeue
+        Map<byte[], AionBlock> blocks;
```

Comment:
Actually, I use FastByteComparisons.equal directly, since I do in-fact need byte[] equality 
(since the block object retrieved from DB is a different object than the one I have in the blocks array)